### ISSUE TITLE
Multipart uploads do not properly handle UTF-8 encoded names

### ISSUE THREAD

**Issue by badvision** [adobe/aem-testing-clients#76]
https://github.com/adobe/aem-testing-clients/blob/889e1c6fc66ee146c5fa1b526065e36082fc480a/src/main/java/com/adobe/cq/testing/client/CQAssetsClient.java#L394

When building the multipart request, the charset of the filename part is not set as UTF-8 and instead is ISO-8859-1 and the filename characters that are 3-byte UTF-8 (such as Korean and Chinese glyphs) are squashed to ? characters.

**Comment by joerghoh** [comment]
Strings in Java are UTF-8; are you sure that this squashing is not done much earlier? For example if you are on a windows platform and provide the filename via command line.

**Comment by badvision** [comment]
Yes, I'm sure.  It's happening in a CentOS Jenkins environment and on my
Mac.

**Comment by badvision** [comment]
Specifically, the content type for the multipart of filename indicates it is text with ISO-1189-1 encoding, even though the request itself has UTF-8 encoding specified.  Most tests don't pick up on this encoding snafu because lower ascii is the same in both encodings.  Only multi-byte characters in the filename reveal this issue.
